article_store/README.md

```diff
@@ -0,0 +1,120 @@
+# Article Store
+
+An XML data store for the Libero Walking skeleton.
+
+## Endpoints
+`GET`
+
+`/articles`
+
+Returns list of article ids.
+
+```{xml}
+<libero:articles xmlns:libero="http://libero.pub">
+    <libero:article>0065-1737-azm-31-03-0367</libero:article>
+    <libero:article>0034-1234-azm-32-04-0777</libero:article>
+    <libero:article>0067-1777-azm-34-08-0237</libero:article>
+</libero:articles>
+```
+
+Example:
+
+`curl -v http://localhost:8000/articles -X GET`
+
+---------------------
+
+`DELETE`
+
+`/articles/<article_id>`
+
+Deletes an entire article including all versions and content.
+
+Example:
+
+`curl -v http://localhost:8000/articles/0065 -X DELETE`
+
+---------------------
+
+`POST`
+
+`/articles/<article_id>`
```

Comment:
I think this should be `/articles/<article_id>/versions` as otherwise it's not clear what `POST` is creating (could be another characteristic of an article)

Reply:
In general that applies to various URLs later on like the `PUT` or the `DELETE` of a version. It also leaves the space for `GET /articles/<article_id>/versions` to return some kind of lists of versions, that I can't be obtain at the moment.